packages/frontend/src/pages/UserActivity.tsx

```diff
@@ -299,6 +317,36 @@ const UserActivity: FC = () => {
                         )}
                     </Table>
                 </ActivityCard>
+
+                <PostHogFeature flag={'extended-usage-analytics'} match={true}>
+                    <ActivityCard grid="table-dashboard-views">
+                        <Description>Dashboard views (top 20)</Description>
+                        <Table bordered $showFooter={false}>
+                            <thead>
+                                <tr>
+                                    <th>Dashboard name</th>
+                                    <th>Views</th>
+                                </tr>
+                            </thead>
+                            {showTableViews(
+                                'dashboard-views',
+                                data.dashboardViews,
+                            )}
+                        </Table>
+                    </ActivityCard>
+                    <ActivityCard grid="table-chart-views">
+                        <Description>Chart views (top 20)</Description>
+                        <Table bordered $showFooter={false}>
```

Comment:
If we are going to have more than 10 rows, we need to fix the table scroll.

(ignore data since I hacked FE to print the same results multiple times)
https://github.com/lightdash/lightdash/assets/9117144/ee89936d-5f96-49bc-8269-551ec76e3e42

Reply:
good catch

Reply:
Replaced react table with mantine 

[Screencast from 12-10-23 13:09:03.webm](https://github.com/lightdash/lightdash/assets/1983672/80861aec-e2f1-431b-b612-22373a29e359)